examples/config/config.yaml

```diff
@@ -0,0 +1,13 @@
+debugMode: true
+adaptor:
+  host: example.com
```

Comment:
I did this in the documentation files with the examples.

I wanted to have the yaml file serving as a kind of model (for lack of another word) and put there values that are not the defaults. So I set `example.com` instead of `localhost` because I didn't want to give a feeling that you needed to explicitly specify them every time. I further elaborated this concept in the new push.